ext/gd/gd.c

```diff
@@ -57,13 +57,23 @@
 
 #include "gd_compat.h"
 
-#include <gd.h>
-#include <gd_errors.h>
-#include <gdfontt.h>  /* 1 Tiny font */
-#include <gdfonts.h>  /* 2 Small font */
-#include <gdfontmb.h> /* 3 Medium bold font */
-#include <gdfontl.h>  /* 4 Large font */
-#include <gdfontg.h>  /* 5 Giant font */
+#ifdef HAVE_GD_BUNDLED
+# include "libgd/gd.h"
+# include "libgd/gd_errors.h"
+# include "libgd/gdfontt.h"  /* 1 Tiny font */
+# include "libgd/gdfonts.h"  /* 2 Small font */
+# include "libgd/gdfontmb.h" /* 3 Medium bold font */
+# include "libgd/gdfontl.h"  /* 4 Large font */
+# include "libgd/gdfontg.h"  /* 5 Giant font */
+#else
+# include <gd.h>
```

Comment:
Do we still need the angle-bracket includes here? I assume these files will be found in the directories specified in quotes.

(My current build doesn't include these...)

Not that I couldn't be wrong here.

Reply:
As GD can be installed based on the bundled libgd _or_ based on the system installed libgd we must include the angle bracket includes for the latter as quotes handles the former.
In the bug report there is a more detailed explanation on how the preprocessor handles each one of them and @cmb69 has pointed that by simply using the quoted includes it would disable/break the external libgd build.

Reply:
These are required for building against system libgd.

Reply:
The point is that there is no "libgd/gd.h" etc. for system libgd.

Reply:
Got it. Thank you!

Reply:
I didn't notice the `#ifdef` 😭